pkg/util/metric/store.go

```diff
@@ -46,6 +48,9 @@ type MetricStore struct {
 	podContainerNumaMetricMap map[string]map[string]map[string]map[string]MetricData // map[podUID]map[containerName]map[numaNode]map[metricName]data
 	cgroupMetricMap           map[string]map[string]MetricData                       // map[cgroupPath]map[metricName]value
 	cgroupNumaMetricMap       map[string]map[string]map[string]MetricData            // map[cgroupPath]map[numaNode]map[metricName]value
+
+	// some stat datas, which may rely on the raw datas above
+	deviceStatMetricMap map[string]map[string]MetricData // map[deviceName]map[metricName]data
```

Comment:
Don't recommend define raw and stat data seperately. Actually the 'raw data' properties contain rate metrics as well, such as cpi in cpuMetricMap. Just store the rate metrics in deviceMetricMap and update them.